src/lib/transition_frontier_persistence/intf.ml

```diff
@@ -122,23 +121,16 @@ module type S = sig
 
   type t
 
-  type diff
-
-  type diff_hash
-
   type verifier
 
   type root_snarked_ledger
 
   type consensus_local_state
 
-  val create : ?directory_name:string -> logger:Logger.t -> unit -> t
+  val create : ?directory_name:string -> logger:Logger.t -> int -> t
```

Comment:
Can you name this `int` argument? Something like `buffer_capacity`? It's not immediately obvious from this interface what that int would parameterize.